ote/src/cljs/ote/views/admin/monitor.cljs

```diff
@@ -64,10 +65,10 @@
                                               (update comp)))
                      :component-did-update update})))
 
-(defn monitor-main [e! {:keys [page monitor-data] :as app}]
-  (e! (controller/->QueryMonitorReport))
-  (if monitor-data
-    (let [translate-typekw (fn [type]
+(defn monitor-main [e! app]
+  (if (get-in app [:admin :monitor :monitor-data])
+    (let [monitor-data (get-in app [:admin :monitor :monitor-data])
```

Comment:
Two consecutive calls to same function are redundant, should be replaced using either `if-let`or by moving the let of `monitor-data` symbol outside the if so it can refer to `monitor-data` instead of making the same function call.

Reply:
changed